app/assets/javascripts/katello/hosts/host_and_hostgroup_edit.js

```diff
@@ -125,6 +128,7 @@ KT.hosts.onKatelloHostEditLoad = function(){
             });
         });
     });
+    HOST_EDIT_ATTRIBUTES["os"] = _.uniq(os_selected_attributes.concat(HOST_EDIT_ATTRIBUTES["os"]));
```

Comment:
This modifies the global array on every single reload which is not very nice, isn't there a better way of doing this? Some kind of JavaScript app initializer where you can simply add that once and forever?